### PR TITLE
Add new lnst pool

### PR DESCRIPTION
**Pull request by ZiyadAtiyyeh**

### CODE REVIEW

**Comment by roidayan** [comment]
i dont understand why you have commit adding xml and then commit fixing that xml. merge those commits.

File: gen-h-vrt-018/gen-h-vrt-017.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
         <param name="machine_type" value="baremetal" />
     </params>
     <interfaces>
-        <eth id="p1p1" label="uplink">
+        <eth id="enp130s0f0" label="uplink">
```
Comment (roidayan) [inline gen-h-vrt-018/gen-h-vrt-017.xml]: shouldn't be in a different commit as the commit adding the file is also in this pull request.

Comment (ZiyadAtiyyeh) [reply gen-h-vrt-018/gen-h-vrt-017.xml]: No, it is not the same file. This is for gen-h-vrt-018 pool and not for the new added pool for gen-h-vrt-017.
